### DISCUSSION TITLE
Would it be okay to move to Unit 5?

### DISCUSSION THREAD

**Discussion by ragunathjawahar** [mobile-dev-inc/Maestro#156] [Q&A]
I noticed that the codebase uses JUnit 4 for testing, is it okay if I upgrade to Unit 5 or are there reasons why we are sticking to the previous version?

**Comment by ArthurSav** [comment]
Hi, thanks for the suggestion. We're not planning to update to JUnit5 at the moment but we welcome pull requests.

**Reply by ragunathjawahar** [reply]
Thank you, this was resolved in https://github.com/mobile-dev-inc/maestro/pull/185
